src/app/screens/HomeTabs/Settings/Settings.tsx

```diff
@@ -45,7 +46,7 @@ export const Settings = () => {
         <Spacer m={1} />
         <Separator />
 
-        <Button block onPress={() => GlobalStore.actions.auth.signOut()}>
+        <Button block onPress={() => GlobalStore.actions.auth.signOut() as void}>
```

Comment:
**suggestion:** maybe we can use this pattern instead to avoid casting
```suggestion
        <Button block onPress={() => {
        GlobalStore.actions.auth.signOut()
        }}>
```

Reply:
```suggestion
        <Button block onPress={() => void GlobalStore.actions.auth.signOut()}>
```
this is another option too, without casting.

Reply:
🤓 I haven't seen this before- thanks for the suggestion @pvinis